HW05.Rmd

```diff
@@ -0,0 +1,145 @@
+
+#This is how the table looks after tidying!
+knitr::kable(dad_mom_v2)
+```
+
+# 1) The CTRP data
+
+## Importing the data
+
+```{r include=TRUE}
+# I tried to get all files automatically using apply but I failed to change the working directory within the project :/. So I ended up doing it manually:
+AUC_1 <- read.csv("./CTRP_files/AUC_1.csv",h=T)
+AUC_2 <- read.csv("./CTRP_files/AUC_2.csv",h=T)
+Experiment_info <- read.csv("./CTRP_files/Experiment_info.csv",h=T)
+cancer_cell_line_info <- read.csv("./CTRP_files/cancer_cell_line_info.csv",h=T)
+compound_info <- read.csv("./CTRP_files/compound_info.csv",h=T)
+```
+
+## Joining together CTRP data
+
+First, I combined the AUC tables (which have the same columns, so I used bind_rows).
+
+```{r include=TRUE}
+AUC <- bind_rows(AUC_1,AUC_2)
+knitr::kable(head(AUC))
+```
+
+Then, I merged the AUC data with the other data frames.
+
+```{r include=TRUE}
+
+#First I changed the column name of the experiment ID in the "AUC" table so it matched the one from the "Experiment_info" table.
+AUC_2 <- AUC %>% rename(expt_id=experiment_id) 
+
+#Then I merged the data tables based on the common column (I did this subsequently starting from AUC + Experimenta_info, which had in common the column (i.e. variable) "expt_id")
+AUC_experiment <- full_join(AUC_2,Experiment_info,by = "expt_id")
+AUC_experiment_cancer <- full_join(AUC_experiment,cancer_cell_line_info,by = "master_ccl_id")
+AUC_experiment_cancer_compound <- full_join(AUC_experiment_cancer,compound_info,by = "master_cpd_id")
+
+#Just to check the final table column names
+colnames(AUC_experiment_cancer_compound)
+
+#Exported the new data frame, just in case.
+write.csv(AUC_experiment_cancer_compound,"CTRP_data_all.csv")
+```
+
+## Answers to questions
+
+### Which cancer type has the lowest AUC values to the compound "vorinostat"?
+
+Based on the figure **hematopoietic and lymphoid tissue cancer** has the lowest AUC values on average. However, the lowest AUC value correspond to **upper aerodigestive tract cancer**.
+
+```{r include=TRUE}
+#First I filtered the complete date set to include only the observations involving the compound "vorinostat"
+vorinostat_data <- AUC_experiment_cancer_compound %>% filter(cpd_name=="vorinostat")
+
+#Then, I built a boxplot to visualize the difference in AUC among cancer types:
+ggplot(vorinostat_data,mapping = aes(x = cancer_type, y= area_under_curve)) +
+  geom_jitter(col="steelblue4",size=0.8)+
+  geom_boxplot(fill="steelblue4",alpha=0.6,outlier.colour = NA)+
+  theme_bw()+
+  theme(axis.title = element_text(color="black", size=12))+
+  xlab("Cancer type") +
+  ylab("Area under curve (AUC)")+
+  theme(axis.text.x = element_text(color="black", size=8, angle=45,hjust=1))
```

Comment:
Your graph for this is GORGEOUS. I like your customization, especially the addition of the visualization of individual data points.